index/scorch/segment/zap/posting.go

```diff
@@ -533,17 +547,22 @@ func (i *PostingsIterator) nextBytes() (
 // nextDocNum returns the next docNum on the postings list, and also
 // sets up the currChunk / loc related fields of the iterator.
 func (i *PostingsIterator) nextDocNum() (uint64, bool, error) {
+	if i.normBits1Hit != 0 {
+		if i.docNum1Hit == math.MaxUint64 {
+			return 0, false, nil
+		}
+		docNum := i.docNum1Hit
+		i.docNum1Hit = math.MaxUint64 // consume our 1-hit docNum
```

Comment:
quick comment, does it make sense to name this math.MaxUint64 to a more readable const like docnumconsumed or you know the names.. :)